src/test/expand.test.ts

```diff
@@ -136,6 +136,9 @@ describe('Expand Abbreviations', () => {
 	// `output.reverseAttributes` emmet option
 	testExpand('html', 'a.dropdown-item[href=#]{foo}', '<a href="#" class="dropdown-item">foo</a>', { "output.reverseAttributes": false });
 	testExpand('html', 'a.dropdown-item[href=#]{foo}', '<a class="dropdown-item" href="#">foo</a>', { "output.reverseAttributes": true });
+
+  	// https://github.com/microsoft/vscode/issues/114923
+	testExpand('html', 'figcaption', '<figcaption></figcaption>');
```

Comment:
Since we want to make sure that a suggestion shows up for `figcaption`:
```suggestion
	testExpandWithCompletion('html', 'figcaption', '<figcaption></figcaption>');
```